src/backend.cpp

```diff
@@ -6,14 +6,14 @@ See https://github.com/vpisarev/loops/LICENSE
 
 #include "backend.hpp"
 #include "func_impl.hpp"
-
+#include <iostream>
 namespace loops
 {
-Mnemotr::Argutr::Argutr(const Arg& a_fixed) : tag(T_FIXED), fixed(a_fixed) {}
-Mnemotr::Argutr::Argutr(size_t a_src_arnum) : tag(T_FROMSOURCE), srcArgnum(a_src_arnum) {}
+Mnemotr::Argutr::Argutr(const Arg& a_fixed) : tag(T_FIXED), fixed(a_fixed), srcArgnum(-1), transitFlags(0) {}
```

Comment:
please, choose better names. Mnemotr and Argutr should be replaced with something else

Reply:
Done.